redbot/core/events.py

```diff
@@ -244,15 +244,14 @@ async def on_command_error(ctx, error, unhandled_by_cog=False):
                 "Exception in command '{}'".format(ctx.command.qualified_name),
                 exc_info=error.original,
             )
-
-            message = _(
-                "Error in command '{command}'. Check your console or logs for details."
-            ).format(command=ctx.command.qualified_name)
             exception_log = "Exception in command '{}'\n" "".format(ctx.command.qualified_name)
             exception_log += "".join(
                 traceback.format_exception(type(error), error, error.__traceback__)
             )
             bot._last_exception = exception_log
+
+            message = await bot._config.invoke_error_msg()
+            message.replace("{command}", ctx.command.qualified_name)
             await ctx.send(inline(message))
```

Comment:
```suggestion
            message = await bot._config.invoke_error_msg()
            message = message.replace("{command}", ctx.command.qualified_name)
            await ctx.send(inline(message))
```

`str.replace` returns a copy of the string like many other `str` methods, it doesn't update the actual string. You'd need to reassign this copy to the same variable or pass the copy straight to the function below.